ui.apps/src/main/content/jcr_root/apps/core/cif/clientlibs/common/js/CommerceGraphqlApi.js

```diff
@@ -55,6 +55,9 @@
                     items {
                         sku
                         name
+                        thumbnail {  
```

Comment:
To be be consistent with the variants here, we should use `media_gallery_entries` too, instead of `thumbnail`.

Reply:
`thumbnail` has the advantage of providing the full asset url, `media_gallery_entries` only provide relative file paths.

Reply:
Then we can use `thumbnail` for both. However, we should stay consistent. Either one or the other.

Reply:
👍 for only using one approach

Reply:
👍 for thumbnail on both, then we can get rid of the prefix concatenation

Reply:
To be clear : I am already using thumbnail here. I am updating the code so that variants also use thumbnail. Right? :)

Reply:
@AKumarSingh yes please :)